profile_bluesky/startup/instrument/devices/monochromator.py

```diff
@@ -51,18 +65,40 @@ def get_motor_pv(label):
         # self._z_pv = get_motor_pv("Z")
 
         super().__init__(
-            prefix, limits=limits, readback_pv=readback_pv,
-            setpoint_pv=setpoint_pv, name=name, read_attrs=read_attrs,
+            prefix, limits=limits, name=name, read_attrs=read_attrs,
             configuration_attrs=configuration_attrs, parent=parent, egu=egu,
             **kwargs
         )
 
+        # Make the default alias for the readback the name of the
+        # positioner itself as in EpicsMotor.
+        self.readback.name = self.name
+
+    def _setup_move(self, position):
```

Comment:
How does this differ from the `set()` method?

Reply:
For everything that inherits `ophyd.PositionerBase` the `set()` method is there just for bluesky compatibility, but all the work is done by `move()`: https://github.com/bluesky/ophyd/blob/eb27aa979133193601fdd9e80550b1a53f40b3f3/ophyd/positioner.py#L46

Now, in `ophid.PVPositioner` the `move()` just creates a `MoveStatus` and handles some of the subscriptions. The actual move "trigger" is done by `_setup_move()`: https://github.com/bluesky/ophyd/blob/eb27aa979133193601fdd9e80550b1a53f40b3f3/ophyd/pv_positioner.py#L149

That's why for both this, and the `PVPositionerSoftDone`, most of the action (and stuff I changed) happen in the `_setup_move()` method.